generatebundlefile/hack/generate_bundle.sh

```diff
@@ -37,17 +37,17 @@ function generate () {
 
     cd "${BASE_DIRECTORY}/generatebundlefile"
     ./bin/generatebundlefile --input "./data/input_${version/-}.yaml" \
-			     --key alias/${kms_key}
+                 --key alias/${kms_key}
 }
 
 function push () {
     local version=$1
-
     cd "${BASE_DIRECTORY}/generatebundlefile/output"
-    awsAuth "$ORAS_BIN" push --username AWS --password-stdin \
-            "${REPO}:v${version}-${CODEBUILD_BUILD_NUMBER}" bundle.yaml
-    awsAuth "$ORAS_BIN" push --username AWS --password-stdin \
-            "${REPO}:v${version}-latest" bundle.yaml
+    # Turn off command echo so no awsAuth function execution is shown in build logs
+    set +x
+    "$ORAS_BIN" push --username AWS --password $(awsAuth "ecr-public") "${REPO}:v${version}-${CODEBUILD_BUILD_NUMBER}" bundle.yaml
+    "$ORAS_BIN" push --username AWS --password $(awsAuth "ecr-public") "${REPO}:v${version}-latest" bundle.yaml
+    set -x 
```

Comment:
```suggestion
    awsAuth | "$ORAS_BIN" push --username AWS --password-stdin \
            "${REPO}:v${version}-${CODEBUILD_BUILD_NUMBER}" bundle.yaml
    awsAuth | "$ORAS_BIN" push --username AWS --password-stdin \
            "${REPO}:v${version}-latest" bundle.yaml
```

Reply:
I approved this so we'd be unblocked, but this seems like a better approach. It would also be nice if we used the same auth token for both, but not a huge deal I guess

Reply:
I've re-opened https://github.com/aws/eks-anywhere-packages/issues/185 until we can fix this.